modules/ducks.py

```diff
@@ -41,10 +48,20 @@ def __init__(self, bot, events, exports):
             "help": "Should the bot kick if there's no duck?",
              "validate": Utils.bool_or_none})
 
+        exports.add("channelset", {"setting": "ducks-min-unique",
+                                   "help": "Minimum unique users required to "
+                                           "talk before a duck spawns.",
+                                   "validate": Utils.int_or_none})
+
+        exports.add("channelset", {"setting": "ducks-min-messages",
+                                   "help": "Minimum messages between ducks "
+                                           "spawning.",
+                                   "validate": Utils.int_or_none})
+
         events.on("new.channel").hook(self.new_channel)
 
         events.on("received.message.channel").hook(
-            self.channel_message)
+            self.channel_message, EventManager.PRIORITY_LOW)
```

Comment:
If you change this to PRIORITY_MONITOR commands.py will .eat() command events before it hits this event hook, meaning you don't have to .eat() in your command callbacks